docs/config.toml

```diff
@@ -22,20 +22,20 @@ defaultContentLanguage = "en"
   weight = 2
   [[menu.main]]
     name = "Install Porter"
-    url = "/install"
+    url = "/install/"
```

Comment:
Since we are using hugo to preview locally, this change is not needed. I had mentioned the trailing slash as the reason why the redirects were working differently with nginx vs. hugo but it's not something that we should change. Both hugo and netlify append a slash automatically for us. 

If you do want to clean up the links to make things consistent, let's do that in another PR as it doesn't impact this one.

Reply:
You are right. File reversed. Will revisit the trailing issue within another ticket